### PR TITLE
Replace strchr by strpbrk + some fixes

### PR DESCRIPTION
**Pull request by touatily**
- Replace `strchr` with `strpbrk` where applicable.
- Some fixes

### CODE REVIEW

File: src/filters/inspect.c [hunk 1]
```diff
@@ -4435,7 +4434,8 @@ static GF_Err inspect_process(GF_Filter *filter)
 			ctx->dump_pck = GF_FALSE;
 		}
 	}
-	count = gf_list_count(ctx->src_pids);
+	// count is already set in line 4415
+	// count = gf_list_count(ctx->src_pids);
```
Comment (rbouqueau) [inline src/filters/inspect.c:4438]: comments to remove

**Comment by aureliendavid** [comment]
pushed to [buildbot-pr2729](https://github.com/gpac/gpac/tree/buildbot-pr2729) branch to check that all is ok with buildbot

checks can be seen on the commit page https://github.com/gpac/gpac/pull/2729/commits/1d1e41e91fe3b20ac2ba1f75b731d54d89217155 by clicking the small icon before the commit message (above the branch name)

**Comment by aureliendavid** [comment]
I think it's ok to merge

**Comment by jeanlf** [comment]
thanks!

**Pull request merged**